easybuild/easyconfigs/g/GD/GD-2.69-foss-2018b-Perl-5.28.0.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PerlModule'
+
+name = 'GD'
+version = '2.69'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://github.com/lstein/Perl-GD'
+description = """GD.pm - Interface to Gd Graphics Library"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
Same here, why not use the `GCCcore` subtoolchain instead, as was done in the existing `GD-2.68-GCCcore-6.4.0-Perl-5.26.1.eb`?